spec/Authentication/AuthenticationBehavior.php

```diff
@@ -1,11 +0,0 @@
-<?php
-
-namespace spec\Http\Message\Authentication;
-
-trait AuthenticationBehavior
```

Comment:
if there is a technical problem with it, i agree its ok to duplicate
that method.